app/src/forms/attestations/forestrysectoroperatorscreening/index.js

```diff
@@ -0,0 +1,49 @@
+const path = require('path');
+
+const constants = require('./constants');
+const Models = require('./models');
+
+const CommonDataService = require('../../common/dataService');
+const commonDataService = new CommonDataService(Models.models);
+
+const commonEvents = require('../../common/events');
+
+const commonMiddleware = require('../../common/middleware');
+
+const CommonRouter = require('../../common/router');
+const commonRouter = new CommonRouter(constants.SLUG, commonDataService);
+
+const teamEvents = require('../../teammanagement/events');
+const TeamRouter = require('../../teammanagement/router');
+const teamRouter = new TeamRouter(constants.SLUG);
+
+const AttestationController  = require('../controller').OperationTypesController;
+const AttestationDataService  = require('../dataService').OperationTypesDataService;
+const AttestationEmailService  = require('../emailService').OperationTypesEmailService;
+const AttestationPdfService  = require('../pdfService').PdfService;
+const AttestationRouter = require('../router').OperationTypesRouter;
+
+const dataService = new AttestationDataService(Models.models, constants);
+const emailService = new AttestationEmailService(dataService, path.join(__dirname, 'assets'));
+const pdfService = new AttestationPdfService(dataService, path.join(__dirname, 'assets'));
+const formController = new AttestationController(dataService, emailService, pdfService);
+const formRouter = new AttestationRouter(constants.SLUG, formController, Models.models);
+
+commonRouter.on(commonEvents.STATUS_CREATED, async (statusUpdate) => {
+  emailService.sendStatusAssignmentEmail(statusUpdate);
+});
+
+teamRouter.on(teamEvents.ACCESS_REQUESTED, async (accessRequest) => {
+  emailService.sendAccessRequestedEmail(accessRequest);
+});
```

Comment:
Wonder if event handlers could be imported from another supporting file for separation rather than having in index. Probably not an issue with just 2 but if a lot more start getting added might be cleaner.

Reply:
Yeah. I did some experiments, but I am leaving this for now.  Moving the logic to another class doesn't really clean up much.  I want to see how this builds up and then do some refactoring.  I'm sure there is a completely different approach that will be cleaner, just can't see it right now.  We need to be cognizant of how quickly this can fragment.  i.e. one form wants all N events, another wants N-1 and another wants N-3...